aces/Loris
# [Instrument] Prototype for Rendering Instruments in JSON Format

This isn't meant to be pulled as-is. This is to just show a working demo and was built with information from `bmi` and `tsi`.

To test,

1. `checkout` this branch
1. Navigate to `<loris-host-name>/test_react_instrument`
1. Click "Render"

The page starts with `tsi` data but you should be able to put in any kind of valid instrument JSON and run it. I'll need more instrument data before I can build more of this.

![image](https://cloud.githubusercontent.com/assets/5655961/25727165/55e1d452-315a-11e7-8992-ee4ff7f28d4a.png)

WIP
![image](https://cloud.githubusercontent.com/assets/5655961/25727172/5bccee92-315a-11e7-892e-16368c59bc7e.png)

Multi-page
![image](https://cloud.githubusercontent.com/assets/5655961/25727181/63dc2e4a-315a-11e7-82f6-a0e0cb4c9c68.png)
